src/cli/onefuzz/api.py

```diff
@@ -162,7 +166,7 @@ def delete(self, container: str, filename: str) -> None:
         client = self._get_client(container)
         client.delete_blob(filename)
 
-    def get(self, container: str, filename: str) -> bytes:
+    def get(self, container: str, filename: str) -> Any:
```

Comment:
not fully sure if it right way to do. if there are any suggestions, let me know.

Reply:
Instead of using Any, this is a case where `cast` should be used.

At the end of this function, replace:
```python
        return client.download_blob(filename)
```

with:
```
        downloaded = cast(bytes, client.download_blob(filename))
        return downloaded
```

Additionally, you will need to import `cast` from typing.